### DISCUSSION TITLE
Game fps is too low on low android device

### DISCUSSION THREAD

**Discussion by ThanhEG** [ppy/osu#18259] [Ideas] [answered]
I want to play osu!laser but i only have phone and android tv box and play osu!laser on them with very low fps.  I hope there will be a version of osu!lazer that can work more smoothly on low end android devices

**Comment by peppy** [comment, marked as answer]
Performance and optimisation considerations are an ongoing effort. Especially on android devices, where the hardware can be of varying quality, it may be impossible to run well on some devices. Please be patient and wait for future optimisations, or upgrade your device.
